next.config.js

```diff
@@ -1,5 +1,4 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
-  target: "serverless",
   reactStrictMode: true,
```

Comment:
```suggestion
  reactStrictMode: true,
  swcMinify: true,
```

Would this make it any faster?